plugins/obs-ffmpeg/obs-ffmpeg-mux.c

```diff
@@ -539,8 +542,6 @@ static void replay_buffer_hotkey(void *data, obs_hotkey_id id,
 				 obs_hotkey_t *hotkey, bool pressed)
 {
 	UNUSED_PARAMETER(id);
-	UNUSED_PARAMETER(hotkey);
-	UNUSED_PARAMETER(pressed);
```

Comment:
I don't think these should be removed.  Nothing in the function body has changed.  It looks like this is left over from PR #2150.

Reply:
That's exactly it.

I would argue that the pressed line can be removed as the parameter is used in the function.